apps/frontend/src/pages/archive/archive.page.ts

```diff
@@ -40,9 +48,13 @@ export class ArchivePageComponent implements OnInit {
       this.changeDetectorRef.markForCheck();
     });
 
-    this.news$ = this.currentNewsRange$.pipe(
+    this.allNews$ = this.archiveService.getArchiveNews$(0, 9999);
```

Comment:
@wolfeman95, though on the second thought, I reverted your PR, filtering can't be done like that, this puts heavy load on server. I'll keep your markup and I'll add sending queries from frontend to backend with filter params + pagination

Reply:
I moved filter logic to backend in last commits in master

Reply:
OK sounds good. You should still call this.changePagination(0); in your version of filterNewsBy() there is a bug there